app/repositories/events/EventsRepository.scala

```diff
@@ -30,25 +30,39 @@ import scala.concurrent.Future
 
 trait EventsRepository {
   def save(events: List[Event]): Future[Unit]
-  def fetchEvents(eventType: EventType, venueType: VenueType) : Future[List[Event]]
+
+  def fetchEvents(eventType: Option[EventType.Value] = None,
```

Comment:
(Completely subjective comment so ignore if you want): I really hate seeing the use of Enumeration.Value. I prefer to see the enum imported so we can just say ```def fetchEvents(eventType: Option[EventType]......```